README.md

```diff
@@ -45,3 +45,11 @@ asdf install djinni latest
 ### Windows
 
 Download the [latest released batch file](https://github.com/cross-language-cpp/djinni-generator/releases/latest/download/djinni.bat), copy it where you want and [put the folder on your PATH](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/).
+
+### Conan
+
+The generator is available at [conan-center](https://conan.io/center/djinni-generator) for Windows, Linux & macOS:
+
+```sh
+conan install djinni-generator/0.3.1@
+```
```

Comment:
I would like to add that as a suggestion, but adding a suggestion after \``` turns out to be interesting difficult 

Most. likely, you want to add that as a conan `build_requires` , so tools like CMake can pick up the path to the  generator via `find_program`.

But maybe we wait with that until we add the hello world example and can link to that ?

Reply:
Thats a good point, I'm aware that the generator should be added to a conanfile instead of installing it manually like this. This line of code is kind of "symbolic", I guess? 🤷  I feel like a conan user usually knows how to do this right... Now that I think about this it sounds kind of stupid to add a command here that nobody will ever use... 🙈

Reply:
Then just write something like

```
Most. likely, you want to add that as a conan build_requires , so tools like CMake can pick up the path to the generator via find_program.
```
:-)

and once we have the hello djinni example here, we link to it

Reply:
I've adjusted this. It now features a `[build_requires]` block as install example.